### ISSUE TITLE
[Feature] Option to disable cursor

### ISSUE THREAD

**Issue by sashahilton00** [Igalia/balena-wpe#15]
It would be great to have the option to disable/hide the cursor, or at least only show it when the mouse is plugged in and moving around. When used for a kiosk, most of the time a cursor is unwanted/unnecessary.

Option to disable the cursor (env var), possibly also the option to not fully disable it, but at least auto hide it after X seconds.

**Comment by manuelafm** [comment]
Yes, it looks like something sensible.  We will look at it for the next updates.

Thanks a lot for the feedback!

**Comment by Jonny-Johnson** [comment]
I'm also facing this problem. This is a system cursor, right? Probably no way I can hide it somehow with CSS settings of my page?

**Comment by chadmbrown** [comment]
Any eta on removing the cursor?
